gambling.js

```diff
@@ -179,11 +230,11 @@ function addToGamblingStreaks(currentStreak, highestStreak, oppositeStreak, user
  * @param {String} user - the user to add stats to
  */
 function addToGamblingStats(outcome, amount, user) {
-    var plural = 'Losses';
+    var plural = 'Lost';
     var opposite = 'Won';
 
     if (outcome === 'Won') {
-        plural = 'Wins';
+        plural = 'Won';
```

Comment:
Why set plural to 'Won' when outcome already holds the value 'Won'? You now have two variables holding the same value.